eclipse-scout-core/src/layout/logicalgrid/LogicalGridLayoutConfig.ts

```diff
@@ -102,4 +120,28 @@ export class LogicalGridLayoutConfig implements LogicalGridLayoutConfigModel {
     }
     return new LogicalGridLayoutConfig(layoutConfig);
   }
+
+  static initHtmlEnvChangeHandler(widget: Widget, getter: () => LogicalGridLayoutConfig, setter: (config: LogicalGridLayoutConfig) => void) {
+    if (widget.initialized) {
+      return;
+    }
+    let handler = () => {
+      setter(getter()?.clone()); // Clone will read the new defaults and apply the custom options
+    };
+    HtmlEnvironment.get().on('propertyChange', handler);
+    widget.one('destroy', () => {
+      HtmlEnvironment.get().off('propertyChange', handler);
+    });
+  }
+
+  static prepareSmallHgapConfig(layoutConfig?: ObjectOrModel<LogicalGridLayoutConfig>) {
```

Comment:
Two widgets call ensure() to create the config object, and two other widgets call prepareSmallHgapConfig(). I think it would be better understandable if those two ways were more similar. Some ideas:
1. Add a defaults argument to ensure(). Usage:
`this._setProperty('layoutConfig', LogicalGridLayoutConfig.ensure(layoutConfig, {hgap: HtmlEnvironment.get().smallColumnGap}));`
2. Add a second ensureX() method. Usage:
`this._setProperty('layoutConfig', LogicalGridLayoutConfig.ensureWithSmallHgap(layoutConfig));`
4. Add a fluent method. Usage:
`this._setProperty('layoutConfig', LogicalGridLayoutConfig.ensure(layoutConfig).withSmallHgapGap());`